tracing-opentelemetry-http/src/main/java/io/micronaut/tracing/opentelemetry/instrument/http/server/OpenTelemetryServerFilter.java

```diff
@@ -82,6 +84,15 @@ public Publisher<MutableHttpResponse<?>> doFilter(HttpRequest<?> request, Server
         Context newContext = instrumenter.start(parentContext, request);
 
         return OpenTelemetryPublisherUtils.createOpenTelemetryPublisher(requestPublisher, instrumenter, newContext, request, new OpenTelemetryObserver<MutableHttpResponse<?>>() {
+
+            @Override
+            public void doOnNext(MutableHttpResponse<?> object, Context context) {
+                OpenTelemetryObserver.super.doOnNext(object, context);
+                if (object.status().getCode() >= 400 ){
```

Comment:
```suggestion
                if (object.status().getCode() >= 400) {
```

Checkstyle fails because of this line.